src/careamics/dataset/dataset_utils/dataset_utils.py

```diff
@@ -121,3 +121,69 @@ def compute_normalization_stats(image: np.ndarray) -> Tuple[np.ndarray, np.ndarr
     # Define the list of axes excluding the channel axis
     axes = tuple(np.delete(np.arange(image.ndim), 1))
     return np.mean(image, axis=axes), np.std(image, axis=axes)
+
+
+def update_iterative_stats(
+    count: np.ndarray, mean: np.ndarray, m2: np.ndarray, new_values: np.ndarray
+) -> Tuple[np.ndarray, np.ndarray, np.ndarray]:
+    """Update the mean and variance of an array iteratively.
```

Comment:
Method should be referenced in docstring or at least comments.

Reply:
Agree, there should be mention of Welford algorithm and maybe a link to the wiki page (https://en.wikipedia.org/wiki/Algorithms_for_calculating_variance)